enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -448,15 +293,17 @@ N/A
 
 ## Drawbacks
 
-N/A
+TODO
```

Comment:
Probably need to remark the complexity of implementing a CSI driver, delivering a driver via the storage operator.